internal/session/connection.go

```diff
@@ -44,6 +44,8 @@ type Connection struct {
 	UpdateTime *timestamp.Timestamp `json:"update_time,omitempty" gorm:"default:current_timestamp"`
 	// Version of the connection
 	Version uint32 `json:"version,omitempty" gorm:"default:null"`
+	// ConnectionStatus is a field derived from connected_time_range
+	ConnectionStatus string `json:"connection_state,omitempty" gorm:"-"`
```

Comment:
Should the json be `connection_state` when the field is `ConnectionStatus`?

Also, can this just be `Status` since the type already is `Connection`?

Reply:
Sure!